libtcmu_log.c

```diff
@@ -349,13 +370,19 @@ static struct log_buf *tcmu_log_initialize(void)
 	pthread_cond_init(&logbuf->cond, NULL);
 	pthread_mutex_init(&logbuf->lock, NULL);
 
+	darray_init(logbuf->outputs);
+
+	add_output_to_syslog(TCMU_LOG_INFO, NULL);
+	add_output_to_stdout(TCMU_LOG_DEBUG);
+	add_output_to_file(TCMU_LOG_DEBUG, LOG_FILE_PATH);
```

Comment:
Some sort of error message if these fail, so we know something went really wrong and we are not logging anything.